File: dagger/src/main/java/com/dagger/mvp/view/MVPView.java
package com.dagger.mvp.view;

import com.dagger.mvp.model.MVPBean;

import java.util.List;

/**
 * by y on 2017/5/31.
 */

public interface MVPView {
    void showProgress();

    void hideProgress();

    void onNetError();

    void onNetSuccess(List<MVPBean> list);
}

File: UtilsSample/Dagger/src/main/java/sample/util/develop/android/dagger/mvp/view/MVPView.kt
package sample.util.develop.android.dagger.mvp.view


import sample.util.develop.android.dagger.mvp.model.MVPBean

/**
 * by y on 2017/5/31.
 */

interface MVPView {
    fun showProgress()

    fun hideProgress()

    fun onNetError()

    fun onNetSuccess(list: List<MVPBean>)
}
